### PR TITLE
Use user date/time format settings & co.

### PR DESCRIPTION
**Pull request by kornelijepetak**
⚠️ Easier to review commit by commit, but feel free to be brave otherwise.

Fixes

* [x] Apply user's settings for formatting dates and time

of #2040 

Also, fixes:

* [x] Running time entry now shows correct dialog size when `Stop` tab is shown
* [x] `Stop` copy is used instead of `Set` to mimic iOS version
* [x] Correct icons are used on the popup

### CODE REVIEW

File: Toggl.Giskard/Fragments/SelectTimeFragment.cs [hunk 1]
```diff
@@ -98,7 +95,7 @@ public void OnTabSelected(TabLayout.Tab tab)
 
         private void onTabChange(int tabPosition)
         {
-            currentPosition = tabPosition;
+			ViewModel.CurrentTab = tabPosition;
```
Comment (willsb) [inline Toggl.Giskard/Fragments/SelectTimeFragment.cs]: Are those CurrentTabs and not CurrentSpaces?

Comment (willsb) [reply Toggl.Giskard/Fragments/SelectTimeFragment.cs]: ➡️ 🚪

File: Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs [hunk 2]
```diff
@@ -14,20 +15,33 @@ namespace Toggl.Foundation.MvvmCross.ViewModels
...
+        public const int StopTimeTab = 1;
+        public const int DurationTab = 2;
+
         private readonly IMvxNavigationService navigationService;
         private readonly ITimeService timeService;
+        private IDisposable timeServiceDisposable;
 
         private readonly TimeSpanToDurationValueConverter durationConverter = new TimeSpanToDurationValueConverter();
 
+        private readonly DateTimeOffsetTimeFormatValueCombiner timeFormatValueCombiner;
```
Comment (willsb) [inline Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs]: These are never used

Comment (willsb) [reply Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs]: And even if they were, why on earth would you need a converter in a ViewModel?

Comment (kornelijepetak) [reply Toggl.Foundation.MvvmCross/ViewModels/SelectTimeViewModel.cs]: This is from before I got enlightened yesterday by my friend @willsb  😄

**Pull request merged**